src/components/game-room/boardMap/bubble/BubbleContent.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useRef, useState } from 'react';
+
+import Image from 'next/image';
+
+import { cn } from '@/lib/utils';
+import { useSoundEventStore } from '@/stores/useSoundEventStore';
+import { useGameDiceStore } from '@/stores/websocket/useGameDiceStore';
+
+const BubbleContent = ({ isActive }: { isActive: boolean }) => {
```

Comment:
```suggestion

interface BubbleContentProps {
  isActive : boolean
}
const BubbleContent = ({ isActive }: BubbleContentProps) => {
```

와 같이 통일을 하면 좋을 것 같습니다.